source/local/local.go

```diff
@@ -217,30 +248,49 @@ func parseFeatures(lines [][]byte, fileName string) map[string]string {
 
 			key := lineSplit[0]
 
-			// Check if it's a boolean value
-			if len(lineSplit) == 1 {
-				features[key] = "true"
+			if !parsingOpts.SkipFeature {
+				updateFeatures(features, key, lineSplit)
+			} else {
+				delete(features, key)
+			}
+
+			if !parsingOpts.SkipLabel {
+				updateFeatures(labels, key, lineSplit)
 			} else {
-				features[key] = lineSplit[1]
+				delete(labels, key)
 			}
+			// SkipFeature and SkipLabel only take effect for one feature
+			parsingOpts.SkipFeature = false
+			parsingOpts.SkipLabel = false
 		}
 	}
 
-	return features
+	return features, labels
+}
+
+func updateFeatures(m map[string]string, key string, lineSplit []string) {
+	// Check if it's a boolean value
+	if len(lineSplit) == 1 {
+		m[key] = "true"
```

Comment:
`key` is redundant arg

```suggestion
func updateFeatures(m map[string]string, lineSplit []string) {
	key := lineSplit[0]
	// Check if it's a boolean value
	if len(lineSplit) == 1 {
		m[key] = "true"
```